docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -1,197 +1,12 @@
-[[synthetic-run-tests]]
-= Create a browser monitor
+// [discrete]
```

Comment:
I assume we want to just delete this file as well

Reply:
I also haven't found a place for "Run on CI" (below this comment). Do you have thoughts on where this could fit?

Reply:
Yes, this is specific to project monitors, but I wouldn't put it under "Get Started". I think we ultimately need a new top-level item for this entire flow. It could be a sort of minimal stub with the content we have now, then expanded later.

Reply:
@andrewvc what do you think about adding this to the "Manage monitors" page in a "Implement best practices for projects" section (see the [preview](https://observability-docs_2353.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-manage-monitors.html))? The goal of this page being how to manage monitors in the long term _after_ you've initially set up / configured them. If this approach looks ok to you for now, then I think this is ready to go!